lib/mappings/createNew.js

```diff
@@ -1,7 +1,38 @@
-import { createWallet } from '../actions';
+import assert from 'bsert';
+import { selectWallets, selectCurrentChain, selectNetwork } from './selectors';
+import { createWallet, createAccount } from '../actions';
+import { getxpub } from '../utilities';
+
+/*
+ * getxpub then create account using the xpub
+ */
+
+async function createHardwareWatchOnlyAccount(walletId, accountId, options) {
```

Comment:
Probably related to the other discussion about mappings, but any reason this isn't part of an action as this is an action creator?

Reply:
Moved this into the actions directory